src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -214,11 +211,11 @@ def kubernetes_exception_handler(ex, fault_type, summary, error_message='Error o
             logger.warning(message_for_not_found)
         if raise_error:
             telemetry.set_exception(exception=ex, fault_type=fault_type, summary=summary)
-            raise CLIError(error_message + "\nError Response: " + str(ex.body))
+            raise CLIInternalError(error_message + "\nError Response: " + str(ex.body))
```

Comment:
CLIInternalError does not fall under user faults. Is it okay to use it here?

Reply:
There was no user_fault raised here before. So, Its okay I guess.